test/autoCompletion.test.ts

```diff
@@ -1710,5 +1710,109 @@ suite('Auto Completion Tests', () => {
         );
       });
     });
+    describe('Array completion', () => {
+      //test_simpleArrayObject
+      // it('Simple array object completion without "-" without any item', async () => {
+      //   const schema = require(path.join(__dirname, './fixtures/testArrayCompletionSchema.json'));
+      //   languageService.addSchema(SCHEMA_ID, schema);
+      //   const content = 'test_simpleArrayObject:\n  ';
+      //   const completion = parseSetup(content, content.length);
+      //   completion.then(function (result) {
+      //     assert.equal(result.items.length, 1);
+      //     assert.equal(result.items[0]?.label, '- (array item)');
+      //   });
+      // });
```

Comment:
This file contains a lot of commented lines, can you delete them?

Reply:
or if it is a failing test, get it to pass?

Reply:
it fails for now, it should be fixed in the future

```
test_simpleArrayObject:
  #not ok when there is no '-' symbol (should suggest '- (array item)')
```

I removed them. Doesn't make sense to be there...
thanks guys